zxlive/edit_panel.py

```diff
@@ -106,3 +106,7 @@ def _input_circuit(self) -> None:
             cmd = UpdateGraph(self.graph_view, new_g)
             self.undo_stack.push(cmd)
             self.graph_scene.select_vertices(new_verts)
+
+    def shutdown(self):
```

Comment:
Why do you need this method here in `edit_panel` if it is already implemented in `base_panel`?